docs/install.md

```diff
@@ -273,6 +273,12 @@ The default value is `false`, which causes Tekton to override the working direct
 for each `Step` that does not have its working directory explicitly set with `/workspace`.
 For more information, see the [associated issue](https://github.com/tektoncd/pipeline/issues/1836).
 
+- `enable-ready-annotation-on-pod-create`: set this flag to `"true"` to allowq the
+Tekton controller to set the `tekon.dev/ready` flag at pod creation time for 
```

Comment:
suggest "annotation" in place of "flag" here.

Reply:
fixed